Stop place import script

culka opened the pull request:
Basic stop place import script for testing

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)
<!-- Reviewable:end -->

culka committed:
Add basic script to import stop places from jore3

culka committed:
Add basic script to import stop places from jore3

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+             'x-hasura-admin-secret': secret}
+    response = requests.post(graphql, headers=headers, json={"query": query})
+    print(response)
+    if (json.loads(response.content)['data']):
```
konvertoi vastaus vain kerran jsoniksi, mihin requests-kirjastosta löytyy suoraan metodi: `response.json()` ja suosittelen early exittiä sen sijaan että mennään koko ajan sisennystasoja syvemmäksi

```
json_data = response.json()
if not json_data['data]:
    return {}

stop_points = json_data['data']['service_pattern_scheduled_stop_point']
...
```

culka replied:
Done.
<!-- Reviewable comment -OLZlaBq6oHaDyZPonkL:b-896fix -->
<!-- Sent from Reviewable.io -->

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+        for v in values:
+            result_dict.setdefault(v['label'], []).append(v)
+        return result_dict
+    return ''
```
tyhjä dict on myös falsy pythonissa:

```
>>> bool({})
False

```

culka replied:
Done.
<!-- Reviewable comment -OLZgQih6RDLhXz9coHV:b-896fix -->
<!-- Sent from Reviewable.io -->

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+            'validity_end': x['validity_end']
+            } for x in stop_points]
+        for v in values:
+            result_dict.setdefault(v['label'], []).append(v)
```
Tää on (mielestäni) funktionaalisesti sama asia, kun rivien 108 - 117 toteutus, eikä looppaile kahdesti listaa läpi ja tallentele väliaikaista listaa:

```
for x in stop_points:
    result_dict.setdefault(x['label'], []).append({
            'label': x['label'],
            'lat': x['measured_location']['coordinates'][1],
            'lon': x['measured_location']['coordinates'][0],
            'priority': str(x['priority']),
            'validity_start': x['validity_start'],
            'validity_end': x['validity_end']
    })
```

culka replied:
Done.
<!-- Reviewable comment -OLZmDNODb_o6DXaExTU:b-896fix -->
<!-- Sent from Reviewable.io -->

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+            for row in cursor:
+                tulos.append(row)
+
+    if (tulos):
```
pythonissa ei käytetä sulkeita if:n expression ympärillä eli pitäisi olla

```
if tulos:
    ...
```

ja tässä tosiaan ihan turha iffi, kun `return tulos` korvaa rivit 39 - 42 ilman että toiminnallisuus muuttuu (`''` on falsy)

culka replied:
Poistettu muutenkin generaattorimuutoksen myötä
<!-- Reviewable comment -OLZelPh4yAp0o9xAjoO:b-3u52w0 -->
<!-- Sent from Reviewable.io -->

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+            for row in cursor:
+                tulos.append(row)
+
+    if (tulos):
```
rivit 85 - 88 voidaan korvata yhdellä returnilla: `return tulos`

culka replied:
Done.
<!-- Reviewable comment -OLZfy8O14lg1Ti5gjmu:b-896fix -->
<!-- Sent from Reviewable.io -->

janneronkko commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+                validityEnd = stopPoint['validity_end']
+                quayInput.append(quayInputForJore3Stop(stop, stopPoint['label'], validityStart, validityEnd , lon, lat))
+                latCoords.append(lat)
+                lonCoords.append(lon)
```
tekisin ehkä vaan yhen listan coords, jossa lat ja lon ois tuplena eli tässä koodi ois:

```
coords.append((lat, lon))
```

tai käyttäis esim. named-tuplea tai dataclassia (https://docs.python.org/3/library/dataclasses.html) jotta saa coordinate-tyypille järkevät attribuuttien nimet

culka replied:
Jätin nämä nyt ennalleen lopun keskiarvon laskemista varten.
<!-- Reviewable comment -OLZlS2B96MsGGief0Fj:b-r79ayd -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+
+import sys
+
+if sys.version_info[0] != 3:
```
Done.
<!-- Reviewable comment -OLZX0Hh87hLClvy9uiE:b-896fix -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+jore3DatabaseUrl = "localhost:1433"
+jore3DatabaseName = "jore3testdb"
+
+def get_jore3_stops_for_area(pysakki_alue):
```
Tulipa opiskeltua miten nämä toimivat, mutta nyt taitaa olla generaattorina.
<!-- Reviewable comment -OLZeJg0DCybo5v8mLIM:b-syw9xa -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+            inner join jr_lij_pysakkialue pa on (p.pysalueid = pa.pysalueid)
+            inner join jr_esteettomyys e on (p.soltunnus = e.tunnus)
+            inner join jr_varustelutiedot_uusi vt on (p.soltunnus = vt.tunnus)
+            where p.pysalueid = '{pysakki_alue}'
```
Done.
<!-- Reviewable comment -OLZejRO4mk8zJPrDI1w:b-896fix -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+    if (tulos):
+        return tulos
+
+    return ''
```
Done.
<!-- Reviewable comment -OLZfhXt3Q-tKUyYnYmf:b-896fix -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+
+    return ''
+
+def get_jore3_stop_areas():
```
Toteutettu
<!-- Reviewable comment -OLZfi6U905_QUhtch6x:b-9zoqxs -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+        
+        with conn.cursor(as_dict=True) as cursor:
+
+            cursor.execute(f"""select pa.nimi, pa.pysalueid, s.sollistunnus, s.solkirjain from jr_pysakki p
```
Done.
<!-- Reviewable comment -OLZfjbv8vXqjztRDMdv:b-896fix -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+    return tulos
+
+
+def get_jore3_info_spots(stopletter, stopid):
```
Poistettu
<!-- Reviewable comment -OLZlYBj-wu1uEPHasS8:b-o59ca2 -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+            cursor.execute(f"""select ip.* from jr_pysakki p
+            inner join jr_solmu s on (p.soltunnus = s.soltunnus)
+            inner join jr_infopaikka ip on (ip.tunnus = p.soltunnus)
+            where s.solkirjain = '{stopletter}' and s.sollistunnus = '{stopid}'
```
Done.
<!-- Reviewable comment -OLZfyRaClf6z05EI2j3:b-896fix -->
<!-- Sent from Reviewable.io -->

culka commented on importer.py:
```diff
@@ -0,0 +1,417 @@
+
+for stopArea in stopAreas:
+    areaStops = get_jore3_stops_for_area(stopArea['pysalueid'])
+    if (areaStops):
```
Muutettu generaattoriksi
<!-- Reviewable comment -OLZl8KRA6ZzYCur07Sm:b-f0terw -->
<!-- Sent from Reviewable.io -->

culka commented on README.md:
```diff
@@ -531,3 +531,21 @@ If a test case fails because the `com.microsoft.sqlserver.jdbc.SQLServerExceptio
+
+### How to use
+
+Modify the `importer.py` script by redefining the source database details and the target graphql address and admin secret. By default the script runs from the local jore3 test database and uses the base local Jore4 Hasura instance as the target.
```
Nyt arvot luetaan `.env` tiedostosta.
<!-- Reviewable comment -OL_-kCVB3RHIIKgLxAB:bnu4rc4 -->
<!-- Sent from Reviewable.io -->

culka reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 0 of 3 files reviewed, 16 unresolved discussions (waiting on @janneronkko)



<!-- Sent from Reviewable.io -->

jarkkoka reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 0 of 4 files reviewed, 17 unresolved discussions (waiting on @janneronkko)

___
*[`README.md` line 550 at r4](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OLnUfPSAtCscU5j0gtC:-OLnUfPSAtCscU5j0gtD:b-xhjli9) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/5ba22aef32da7b0cc4382202558908d02360a312/README.md#L550)):*
> ```Markdown
> GRAPHQL_SECRET=
> JORE3_USERNAME=
> JORE3_PASSWORD=
> ```

The `README.md` doesn't tell what are the database preconditions for running the Python script.


<!-- Sent from Reviewable.io -->

jarkkoka reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 0 of 4 files reviewed, 19 unresolved discussions (waiting on @culka and @janneronkko)

___
*[`README.md` line 542 at r4](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OLrSSrg0XQ3Vdk-G6ML:-OLrSSrg0XQ3Vdk-G6MM:bh15q4i) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/5ba22aef32da7b0cc4382202558908d02360a312/README.md#L542)):*
> ```Markdown
> 
> By default the script runs from the local jore3 test database and uses the base local Jore4 Hasura instance as the target.
> You can change the source database and target Hasura instance by creating a `.env` file in the same directory as the script.
> ```

Documentation is needed on what are the compatible versions of `jore4-hasura' and other microservices. Actually, I think it would be better to make changes to the `docker-compose.custom.yml` file.
___
*[`README.md` line 544 at r4](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OLrSuPt1PZwyWfPN_Op:-OLrSuPt1PZwyWfPN_Oq:bw89wvb) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/5ba22aef32da7b0cc4382202558908d02360a312/README.md#L544)):*
> ```Markdown
> You can change the source database and target Hasura instance by creating a `.env` file in the same directory as the script.
> 
> Set the values for variables you want to set:
> ```

It is necessary to state how the data is expected to be exported to Azure. By using Azure's Hasura endpoint directly or taking a snapshot of a local database. If the latter is the case, what is the required `pg_dump` command?


<!-- Sent from Reviewable.io -->

culka reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 0 of 6 files reviewed, 19 unresolved discussions (waiting on @janneronkko and @jarkkoka)

___
*[`README.md` line 542 at r4](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OLrSSrg0XQ3Vdk-G6ML:-OM6k9h21fYeBQePTak7:bb71z0i) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/5ba22aef32da7b0cc4382202558908d02360a312/README.md#L542)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Documentation is needed on what are the compatible versions of `jore4-hasura' and other microservices. Actually, I think it would be better to make changes to the `docker-compose.custom.yml` file.
</blockquote></details>

Updated the file and startup script
___
*[`README.md` line 550 at r4](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OLnUfPSAtCscU5j0gtC:-OM6qI5nDN9DQNUHLKQy:b-8b11j9) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/5ba22aef32da7b0cc4382202558908d02360a312/README.md#L550)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

The `README.md` doesn't tell what are the database preconditions for running the Python script.
</blockquote></details>

Added


<!-- Sent from Reviewable.io -->

culka committed:
Dockerize the stop import script

culka committed:
Dockerize the stop import script

jarkkoka reviewed (changes requested):
> Reviewed 2 of 3 files at r5, 1 of 5 files at r6.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 3 of 8 files reviewed, 22 unresolved discussions (waiting on @culka and @janneronkko)

___
*[`stopimport/Dockerfile` line 11 at r6](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMQcx4sDt8B26FoWTZg:-OMQcx4sDt8B26FoWTZh:b-l8ih27) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/0608f3895628bf11e4bfa72c4fd235a7f5909522/stopimport/Dockerfile#L11)):*
> ```Dockerfile
> COPY .env .
> 
> CMD ["python", "importer.py"]
> ```

Line break missing at the end
___
*[`stopimport/run-stopimport.sh` line 1 at r6](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMQdWWl6B4esKSRCpKk:-OMQdWWl6B4esKSRCpKl:b-knk1m9) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/0608f3895628bf11e4bfa72c4fd235a7f5909522/stopimport/run-stopimport.sh#L1)):*
> ```Shell
> #!/usr/bin/env bash
> ```

The containing directory should be renamed from `stopimport` to `stop_registry_import` (note the underscores to improve reading) since plain "stopimport" is misleading, since the actual Java application also imports stops. We need to make it clearer what this Python application does.

In addition, I think this script should be renamed to `run-stop-reg-import.sh`.
___
*[`stopimport/run-stopimport.sh` line 3 at r6](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMQeB6b4CcJ3T57CiLl:-OMQeB6b4CcJ3T57CiLm:b-x40qll) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/0608f3895628bf11e4bfa72c4fd235a7f5909522/stopimport/run-stopimport.sh#L3)):*
> ```Shell
> #!/usr/bin/env bash
> 
> docker build -t stop-importer .
> ```

Rename `stop-importer` to `stop-reg-importer`. We need to make it clearer what this Docker container does, since the actual Java application also imports stops. We don't want to confuse new developers coming to this project in the future.


<!-- Sent from Reviewable.io -->

jarkkoka reviewed (changes requested):
> Reviewed 1 of 6 files at r7.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 3 of 9 files reviewed, 24 unresolved discussions (waiting on @culka and @janneronkko)

___
*[`README.md` line 537 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR76RrBqUBCLBk2Dfg:-OMR76RrBqUBCLBk2Dfh:b-f06zd0) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L537)):*
> ```Markdown
> ---
> 
> ## Jore3 stop import script
> ```

Use "Jore 3" spelling here as is used consistently in this README.
___
*[`README.md` line 545 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR7Nnf7ugaWDSXkW15:-OMR7Nnf7ugaWDSXkW16:bomllsp) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L545)):*
> ```Markdown
> ### How to use
> 
> To run the stop registry importer script you need to have a Jore3 database, a populated Jore4 routes database, Jore4 Hasura and Jore4 Tiamat running.
> ```

Let's fix the Jore spelling to be consistent with the rest of this README file.

Change this sentence to the form:

"To run the stop register importer script, you must have a Jore 3 database, a populated Jore 4 routes database, and the `jore4-hasura` and `jore4-tiamat` microservices running."
___
*[`README.md` line 561 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR8Fia6HcINnF7PTN0:-OMR8Fia6HcINnF7PTN1:b-vcty7n) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L561)):*
> ```Markdown
> ```
> 
> You should have run the base Jore3 importer first which ensures the Jore4 database has the required scheduled stop points. Then run the stop registry import script which will match scheduled stop points in the Jore4 routes database with stops in the Jore3 database and generate GraphQL mutations to Hasura/Tiamat according to the data. The script will also link the generated stop registry stops with the scheduled stop points by their NeTEx ID using Hasura for the mutation.
> ```

Let's be consistent in Jore spelling with the rest of the README file:
- `Jore3` => `Jore 3`
- `Jore4` => `Jore 4`


<!-- Sent from Reviewable.io -->

jarkkoka reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 3 of 9 files reviewed, 25 unresolved discussions (waiting on @culka and @janneronkko)

___
*[`stop-registry-importer/importer.py` line 35 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMRAJEm1TYJcIZukFPu:-OMRAJEm1TYJcIZukFPv:b71h6lb) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/stop-registry-importer/importer.py#L35)):*
> ```Python
>         with conn.cursor(as_dict=True) as cursor:
> 
>             cursor.execute(f"""select * from jr_pysakki p
> ```

The SQL query does not conform to the general SQL style guide used e.g. in the `jore4-hasura` repository where SELECT, FROM and other such keywords are in uppercase.


<!-- Sent from Reviewable.io -->

jarkkoka reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 3 of 9 files reviewed, 27 unresolved discussions (waiting on @culka and @janneronkko)

___
*[`stop-registry-importer/importer.py` line 51 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMRRHUkBn88cQ5xwopI:-OMRRHUkBn88cQ5xwopJ:b-5dzqz2) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/stop-registry-importer/importer.py#L51)):*
> ```Python
>         with conn.cursor(as_dict=True) as cursor:
> 
>             cursor.execute("""select pa.nimi, pa.pysalueid, s.sollistunnus, s.solkirjain from jr_pysakki p
> ```

For better reading, `FROM` should start from a new line.
___
*[`stop-registry-importer/importer.py` line 55 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMRRO5CDIDjvj1p0R2U:-OMRRO5CDIDjvj1p0R2V:bp13rp8) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/stop-registry-importer/importer.py#L55)):*
> ```Python
>             inner join jr_lij_pysakkialue pa on (p.pysalueid = pa.pysalueid)
>             where p.pysalueid in (
>             select jp.pysalueid from jr_pysakki jp
> ```

For better reading, `FROM` should start from a new line


<!-- Sent from Reviewable.io -->

culka committed:
Fetch all stop data at once in stop registry importer

culka committed:
Fetch all stop data at once in stop registry importer

culka committed:
Insert HSL zones in startup script

culka committed:
Insert HSL zones in startup script

culka reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151)* status: 1 of 10 files reviewed, 27 unresolved discussions (waiting on @janneronkko and @jarkkoka)

___
*[`README.md` line 537 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR76RrBqUBCLBk2Dfg:-OMvr_hyEnWhlYNyyIvV:b-ds5o1f) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L537)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Use "Jore 3" spelling here as is used consistently in this README.
</blockquote></details>

Both spellings seem to be used in this file though?
___
*[`README.md` line 545 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR7Nnf7ugaWDSXkW15:-OMvt5GS6yStR4ZGq539:b-896fix) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L545)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Let's fix the Jore spelling to be consistent with the rest of this README file.

Change this sentence to the form:

"To run the stop register importer script, you must have a Jore 3 database, a populated Jore 4 routes database, and the `jore4-hasura` and `jore4-tiamat` microservices running."
</blockquote></details>

Done.
___
*[`README.md` line 561 at r7](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMR8Fia6HcINnF7PTN0:-OMvuBY01ksvHO2Bq9q1:b-896fix) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/ac6e8e935a99c066976586d54ac2398c54c4164a/README.md#L561)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Let's be consistent in Jore spelling with the rest of the README file:
- `Jore3` => `Jore 3`
- `Jore4` => `Jore 4`
</blockquote></details>

Done.
___
*[`stopimport/Dockerfile` line 11 at r6](https://reviewable.io/reviews/HSLdevcom/jore4-jore3-importer/151#-OMQcx4sDt8B26FoWTZg:-OMvunan0y_IR2tKlvKm:b-7pejw8) ([raw file](https://github.com/HSLdevcom/jore4-jore3-importer/blob/0608f3895628bf11e4bfa72c4fd235a7f5909522/stopimport/Dockerfile#L11)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Line break missing at the end
</blockquote></details>

There is line break at the end?


<!-- Sent from Reviewable.io -->

Huulivoide committed:
Python importer: Fix white space

Huulivoide committed:
Python importer: Fix white space

Huulivoide committed:
Python importer: Store StopArea PrivateCode with type HSL/JORE-3

Huulivoide committed:
Python importer: Store StopArea PrivateCode with type HSL/JORE-3

Huulivoide committed:
Set HSL/JORE-3 PrivateCode type on StopPlace not Quay

Huulivoide committed:
Set HSL/JORE-3 PrivateCode type on StopPlace not Quay

Huulivoide committed:
Python importer: Savce Quay PrivateCode with type HSL/JORE-3

Huulivoide committed:
Python importer: Savce Quay PrivateCode with type HSL/JORE-3

jannebe committed:
Update importer script for new data model

- Save stop details in the stop_place - quay model
- Use requirements.txt to define libraries
- Update README
- Update docker setup

culka committed:
Update importer script for new data model

- Save stop details in the stop_place - quay model
- Use requirements.txt to define libraries
- Update README
- Update docker setup

jannebe committed:
stop-registry-importer Add doc string to importer.py

jannebe committed:
make jore3/jore4 variables more explicit in importer.py

jannebe committed:
upgrade stop registry importer python and libs

jannebe committed:
stop importer: include tram stops

jannebe committed:
stop-registry-importer logging

jannebe committed:
Run stop-registry-importer as last batch job step
